site/src/pages/components/accordion.explainer.mdx

```diff
@@ -289,3 +317,21 @@ while intersecting,
 are different enough
```

Comment:
Is it worth adding a link to https://github.com/openui/open-ui/issues/559 above?

I also don't quite understand the difference between "panelset" and "spicy sections". They both seem like a thing that goes between accordions and tabs. I'm not saying you need to change anything about the explainer, per se, just asking if you agree. If so, I suppose you could add a sentence along those lines. Or combine the two sections together. Or leave them as-is.